app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -24,33 +26,47 @@
 public class LoginViewModel extends ViewModel {
 
     private final AuthService loginModel;
+    private final EncryptionService encryptionService;
     private final Login login = new Login();
     private final HostSelectionInterceptor interceptor;
     private final Preferences sharedPreferenceModel;
     private final CompositeDisposable compositeDisposable = new CompositeDisposable();
+    private final String PREF_USER_PASSWORD = "user_password";
+    private final String PREF_USER_EMAIL = "user_email";
 
     private final MutableLiveData<Boolean> progress = new MutableLiveData<>();
     private final MutableLiveData<String> error = new MutableLiveData<>();
+    private final MutableLiveData<Login>  decryptedLogin = new MutableLiveData<>();
     private MutableLiveData<Boolean> isLoggedIn;
     private final MutableLiveData<Boolean> isTokenSent = new MutableLiveData<>();
     private MutableLiveData<Set<String>> emailList;
 
     @Inject
-    public LoginViewModel(AuthService loginModel, HostSelectionInterceptor interceptor, Preferences sharedPreferenceModel) {
+    public LoginViewModel(AuthService loginModel, HostSelectionInterceptor interceptor, Preferences sharedPreferenceModel, EncryptionService encryptionService) {
         this.loginModel = loginModel;
         this.interceptor = interceptor;
         this.sharedPreferenceModel = sharedPreferenceModel;
+        this.encryptionService = encryptionService;
+    }
+
+    private void encryptUserCredentials() {
+        String encryptedEmail = encryptionService.encrypt(login.getEmail());
+        String encryptedPassword = encryptionService.encrypt(login.getPassword());
+        sharedPreferenceModel.saveString(PREF_USER_EMAIL, encryptedEmail);
+        sharedPreferenceModel.saveString(PREF_USER_PASSWORD, encryptedPassword);
     }
 
     //for logging into the app
     public void login() {
         compositeDisposable.add(loginModel.login(login)
             .doOnSubscribe(disposable -> progress.setValue(true))
             .doFinally(() -> progress.setValue(false))
-            .subscribe(() -> isLoggedIn.setValue(true),
+            .subscribe(() -> {  isLoggedIn.setValue(true);
+                                encryptUserCredentials();
+                             },
```

Comment:
Everything seems good. Just properly format this line and encrypt before sending true